Objects/object.c

```diff
@@ -860,6 +860,12 @@ PyObject_Compare(PyObject *v, PyObject *w)
 {
     int result;
 
+    if (Py_Py3kWarningFlag &&
+        PyErr_WarnEx_WithFix(PyExc_Py3xWarning, "the cmp method is not supported in 3.x",
+        "implement the function to a utility library", 1) < 0) {
```

Comment:
Not sure I can parse the fix. Does it mean something like "Use an external library such as <blah>"?

Reply:
I mean, since `cmp()` is removed in 3.x, the only option is if you want to still use a `cmp()` function call,  implement your own version in a utility library, or rewrite your code to use the newer rich comparison operations.

Reply:
Right. So maybe "You can either provide your own alternative or use the <blah> library" where `<blah>` is some library that offers a backwards compatible fix for `cmp`? That's quite a useful message for most programmers I reckon!